library/std/src/sync/condvar.rs

```diff
@@ -195,8 +195,11 @@ impl Condvar {
         if poisoned { Err(PoisonError::new(guard)) } else { Ok(guard) }
     }
 
-    /// Blocks the current thread until this condition variable receives a
-    /// notification and the provided condition is false.
+    /// Blocks the current thread while the provided condition is satisfied.
```

Comment:
Could you just tweak this to something like "becomes `false`" or "stops returning true" rather than "is satisfied"? Unfortunately it's a bit unintuitive direction (imo) so it would be good to have this clear.

Otherwise looks good, thanks for the changes! r=me after the above

Reply:
Ahhhhh I'm sorry, I didn't think the exact wording through 😢 - "while" + "stops returning" doesn't work here. "Blocks the current thread until the provided condition becomes false." is probably the better option here.

Also missing a period at the end of that sentence.